Deployment.md

```diff
@@ -0,0 +1,44 @@
+# Deployment Practices
+
+> Note: This document covers steps for deploying projects to production, which is only performed
+> by MetaBrainz contractors. It includes discussion about servers and repositories which other
+> contributors won't have access to.
```

Comment:
Since this document is destinated to maintainers only, it should probably live in the syswiki instead, along with the one for MusicBrainz Server’s deployment?

Reply:
we  talked on IRC and decided that this was a good idea, I'll move it.